vllm/model_executor/layers/quantization/modelopt.py

```diff
@@ -1107,7 +1111,9 @@ def apply(
             layer.alpha,
             output_dtype,
         )
-        if self.backend == "flashinfer-trtllm":
+        if self.backend == "flashinfer-cudnn":
+            out = flashinfer_scaled_fp4_mm(*mm_args, backend="cudnn")
```

Comment:
May be also simplify this here? Since all of them just say "flashinfer-<backend>"?